header/index.html

```diff
@@ -415,6 +427,37 @@ <h2 class="mt-5">Without navigation links</h2>
             >
               <div class="h1 my-3"><a href="/">Website at Stanford</a></div>
             </div>
+            <div>
+            <nav
+                class="navbar collapse navbar-collapse"
+                aria-label="browse" id="user-util-collapse"
```

Comment:
The id `user-util-collapse` should only occur once on the page. The aria label should be unique across all the `nav` nodes.

Reply:
It is only showing up once per header. It has always showed up twice on the page because of the two headers.

Reply:
Yes, but it must only exist once in the whole DOM for it to be valid HTML.  Let's fix it now.